baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -50,7 +50,9 @@ Leading and trailing whitespaces (`OWS`) are allowed but MUST be trimmed when co
 
 #### value
 
-A value contains a URL encoded UTF-8 string.
+A value contains UTF-8 encoded string.
+Any characters outside of the `baggage-octet` ranges of characters MUST be percent-encoded.
```

Comment:
iiuc, URL-encoding is potentially a stronger form of percent-encoding for non-`baggage-octet` characters. Many languages gave URL-encoding functions, not so sure about the other one. And if a string is encoded with stronger form, it can always be decoded. So the question is - can we explicitly allow percent-encoding other characters as well, if it's easier for the implementation?

Reply:
Yes we can. the referenced RFC has similar wording.